optimum/habana/distributed/tensorparallel.py

```diff
@@ -7,6 +7,11 @@
 import torch.distributed._functional_collectives as funcol
 from torch import nn
 
+#This needs to be fixed
```

Comment:
may be a JIRA ID will be more informative

Reply:
Done